Tuist/ProjectDescriptionHelpers/Module.swift

```diff
@@ -48,16 +44,16 @@ public enum Module: CaseIterable {
 
     public var testsResources: ProjectDescription.ResourceFileElements {
         [
-            "\(frameworkPath)/\(Constant.testsPath)/**/.gitkeep",
-            "\(frameworkPath)/\(Constant.testsPath)/\(Constant.resourcesPath)/**"
+            "\(frameworkPath)/\(Constant.testsPath)/\(Constant.resourcesPath)/**",
+            "\(frameworkPath)/\(Constant.testsPath)/**/.gitkeep"
         ]
     }
 
-    public func getBundleId(mainBundleId: String) -> String {
-        "\(mainBundleId).\(name)"
+    public func bundleId(mainBundleId: String) -> String {
+        "\(mainBundleId).\(name.lowercased())"
```

Comment:
Regarding [Apple documentation](https://developer.apple.com/documentation/bundleresources/information-property-list/cfbundleidentifier), the bundle identifier supports (A–Z, a–z, and 0–9), hyphens (-), and periods (.). I think we should keep it as it is instead of lowering cases. Please revert for `testBundleId` function also 🙏